### PR TITLE
Removed globals and added configuration options

### PR DESCRIPTION
**Pull request by thomasfr**
You can now configure if sequelize should be exposed globally. By default this is set to default.
To disable:
`config/globals.js` add `sequelize: false`
It also respects the globals configuration for models.

I had to remove the ENUM column in the test fixture model `UserGroup`.

### CODE REVIEW

**Comment by thomasfr** [comment]
I have no idea why the test is failing on travis. The test is green locally:
bildschirmfoto 2015-10-17 um 22 40 57

**Comment by festo** [comment]
The tests are failed because it search `Sequalize` globally and can't find:
https://travis-ci.org/festo/sails-hook-sequelize/builds/85950018

Not all of your cahnges are approved by me, but I can understand the importance of global and not global part ... but untill the test are failed, I won't merge this request.

**Comment by thomasfr** [comment]
The problem was that i tried to require 'Sequelize' instead of lowercased 'sequelize' module. So now it should be fine to merge that.
I also added some npm run scripts to initialize the needed postgres user and database for the tests. You can now run `npm run db:init`. This will create a local postgres datadir and start the postgres server. This way it is easier to develop and contribute :)

**Comment by thomasfr** [comment]
Can i do something to help you to merge this?

**Comment by markelog** [comment]
Why this is still not landed?

**Comment by aquavitae** [comment]
Any progress on the PR?

**Comment by lancedikson** [comment]
@festo, ping

**Comment by KSDaemon** [comment]
Hi @thomasfr and all! I think it's a good idea! Can you sync your codebase with last changes and update PR? 
By the way, I took on the role of a maintainer, so don't let the project die! :)

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,6 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+.idea
```
Comment (KSDaemon) [inline .gitignore]: I don't think it's a good idea to submit to repos gitignore files and dirs that do not belongs to project itself. You can always set them in your user gitignore. Some devs use idea, some vscode, some sublime, etc. Let's keep project clean!

**Comment by KSDaemon** [comment]
Also, it would be cool, if you update docs with examples of new possible configuration!

**Comment by KSDaemon** [comment]
Hi @thomasfr! I've implemented this sort of feature a little bit differently. Feel free to check, test, comment and edit! :)

**Pull request closed without merging**